crypto/src/commitments/kzg.rs

```diff
@@ -323,6 +323,84 @@ mod tests {
         assert!(kzg.verify(&x, &y, &p_commitment, &proof));
     }
 
+    #[test]
+    fn poly_9000_constant_should_verify_proof() {
+        let kzg = KZG::new(create_srs());
+        let p = Polynomial::<FrElement>::new(&[FieldElement::from(9000)]);
```

Comment:
I think we could just write `let p = Polynomial::new(&[FieldElement::from(9000)]);`